test/test_nn.py

```diff
@@ -12366,6 +12366,14 @@ def test_embedding_padding_idx(self, device, dtype):
         self.assertEqual(output[0][2].sum(), 0)
         self.assertEqual(output[1][1].sum(), 0)
 
+        # change padding vector
+        padding_vector = torch.ones(20, dtype=dtype)
+        embedding = nn.Embedding(10, 20, padding_idx=2, sparse=True).to(device, dtype)
+        embedding.weight.data[2] = padding_vector
```

Comment:
nit: you should never use `.data`.
If you want to update the weights without the autograd tracking it, you should use `with torch.no_grad()`